lib/spack/docs/index.rst

```diff
@@ -37,23 +37,32 @@ package:
 If you're new to spack and want to start using it, see :doc:`getting_started`,
 or refer to the full manual below.
 
------------------
-Table of Contents
------------------
 
 .. toctree::
    :maxdepth: 2
+   :caption: Tutorials
 
    features
    getting_started
    basic_usage
-   packaging_guide
-   mirrors
+   workflows
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Reference Manual
+
    configuration
-   developer_guide
-   case_studies
-   command_index
+   mirrors
    package_list
+   command_index
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contributing to Spack
+
+   contribution_guide
```

Comment:
This will cause Sphinx warnings since it hasn't been added to Spack yet. I would remove it for now and wait for @davydden's PR to be merged. If your PR is merged first I'll ask him to add it to his index.rst.